tests/e2e/file_volume_statefulsets.go

```diff
@@ -508,4 +508,175 @@ var _ = ginkgo.Describe("[csi-file-vanilla] File Volume statefulset", func() {
 		gomega.Expect(len(ssPodsAfterScaleDown.Items) == int(replicas)).To(gomega.BeTrue(),
 			"Number of Pods in the statefulset should match with number of replicas")
 	})
+
+	/*
+	   Test performs following operations
+
+	   Steps
+	   1. Create a storage class.
+	   2. Create nginx service.
+	   3. Create nginx statefulsets with 3 replicas.
+	   4. Wait until all Pods are ready and PVCs are bounded with PV.
+	   5. Restart CSI dameonset pods
+	   6. Scale down statefulsets to 2 replicas.
+	   7. Scale up statefulsets to 3 replicas.
+	   8. Scale down statefulsets to 0 replicas and delete all pods.
+	   9. Delete all PVCs from the tests namespace.
+	   10. Delete the storage class.
+	*/
+	ginkgo.It("Statefulset with file volume testing with CSI daemonset restart", func() {
+		curtime := time.Now().Unix()
+		randomValue := rand.Int()
+		val := strconv.FormatInt(int64(randomValue), 10)
+		val = string(val[1:3])
+		curtimestring := strconv.FormatInt(curtime, 10)
+		scName := "nginx-sc-default-" + curtimestring + val
```

Comment:
why are we creating storage class with different name ?

Reply:
Does this cause any regression ??